app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+    def self.search_food(search)
+        data = FoodService.search_food(search)
+        data[:foods].map do |food_data|
+            FoodInfo.new(food_data)
+        end
+    end
```

Comment:
I would probably limit to 10 either here, like `data[:foods].take(10).map ...` OR, some APIs have a way for you to limit the responses that get sent back to you. I believe the USDA has a parameter called `limit` you could use to automatically limit your responses to 10.